### PR TITLE
old world users rejected agreement within 10months

### PR DESCRIPTION
**Pull request by takanocap**

### CODE REVIEW

File: helm/ffc-ahwr-document-generator/templates/config-map.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ data:
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE: {{ quote .Values.container.notifyTemplateIdFarmerApplicationComplete }}
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_NEW_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteNewUser }}
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExisitingUser }}
+  NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER_REJECTED_WITHIN_10_MONTHS: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExistingUserRejectedWithinTenMonths }}
```
Comment (alex742) [inline helm/ffc-ahwr-document-generator/templates/config-map.yaml:15]:
```suggestion
  NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER_REJECTED_WITHIN_10_MONTHS: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExisitingUserRejectedWithin10Months }}
```

10 not Ten

File: helm/ffc-ahwr-document-generator/templates/config-map.yaml [hunk 2]
```diff
@@ -12,6 +12,7 @@ data:
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE: {{ quote .Values.container.notifyTemplateIdFarmerApplicationComplete }}
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_NEW_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteNewUser }}
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExisitingUser }}
+  NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER_REJECTED_WITHIN_10_MONTHS: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExisitingUserRejectedWithin10Months }}
```
Comment (alex742) [inline helm/ffc-ahwr-document-generator/templates/config-map.yaml:15]:
```suggestion
  NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER_REJECTED_WITHIN_10_MONTHS: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExistingUserRejectedWithin10Months }}
```

File: helm/ffc-ahwr-document-generator/templates/config-map.yaml [hunk 3]
```diff
@@ -12,6 +12,7 @@ data:
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE: {{ quote .Values.container.notifyTemplateIdFarmerApplicationComplete }}
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_NEW_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteNewUser }}
   NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExisitingUser }}
```
Comment (alex742) [inline helm/ffc-ahwr-document-generator/templates/config-map.yaml:14]:
```suggestion
  NOTIFY_TEMPLATE_ID_FARMER_APPLICATION_COMPLETE_EXISTING_USER: {{ quote .Values.container.notifyTemplateIdFarmerApplicationCompleteExistingUser }}
```

File: helm/ffc-ahwr-document-generator/values.yaml [hunk 4]
```diff
@@ -44,6 +44,7 @@ container:
   notifyTemplateIdFarmerApplicationComplete:
   notifyTemplateIdFarmerApplicationCompleteNewUser:
   notifyTemplateIdFarmerApplicationCompleteExisitingUser:
```
Comment (alex742) [inline helm/ffc-ahwr-document-generator/values.yaml:46]:
```suggestion
  notifyTemplateIdFarmerApplicationCompleteExistingUser:
```

**Review by Afshin-Karamifar-Capgemini — APPROVED** [review]

**Pull request merged**